### ISSUE TITLE
Error when using the Gallery App slideshow

### ISSUE THREAD

**Issue by enoch85** [owncloud/gallery#137]
```
Fatal	index	Exception: \OC_Image::_output(): "image/x-ms-bmp" is not supported when forcing a specific output format	2014-11-23T21:19:03+00:00
Fatal	index	Exception: \OC_Image::_output(): "image/x-ms-bmp" is not supported when forcing a specific output format	2014-11-23T21:19:02+00:00
Error	PHP	imagecreatefromstring(): one parameter to a memory allocation multiplication is negative or zero, failing operation gracefully at /var/www/cloud.fsgo.se/owncloud/lib/private/image.php#531	2014-11-23T21:17:58+00:00
Error	PHP	filesize(): stat failed for /var/www/cloud.fsgo.se/owncloud/data/admin/gallery/admin//JV ALLA/-=Äldre Ej aktuellt/-=PP (Positiv Pension)/Kundkort - konsument.square.bmp at /var/www/cloud.fsgo.se/owncloud/apps/gallery/lib/thumbnail.php#107	2014-11-23T21:17:58+00:00
Error	PHP	filemtime(): stat failed for /var/www/cloud.fsgo.se/owncloud/data/admin/gallery/admin//JV ALLA/-=Äldre Ej aktuellt/-=PP (Positiv Pension)/Kundkort - konsument.square.bmp at /var/www/cloud.fsgo.se/owncloud/apps/gallery/lib/thumbnail.php#106	2014-11-23T21:17:58+00:00
Error	PHP	imagecreatefromstring(): Couldn't create GD Image Stream out of Data at /var/www/cloud.fsgo.se/owncloud/lib/private/image.php#531	2014-11-23T21:17:58+00:00
Error	PHP	imagecreatefromstring(): Passed data is not in 'WBMP' format at /var/www/cloud.fsgo.se/owncloud/lib/private/image.php#531	2014-11-23T21:17:58+00:00
Fatal	index	Exception: \OC_Image::_output(): "image/x-ms-bmp" is not supported when forcing a specific output format	2014-11-23T21:17:58+00:00
Fatal	index	Exception: \OC_Image::_output(): "image/x-ms-bmp" is not supported when forcing a specific output format	2014-11-23T21:17:57+00:00
Fatal	index	Exception: \OC_Image::_output(): "image/x-ms-bmp" is not supported when forcing a specific output format	2014-11-23T21:17:56+00:00
Fatal	index	Exception: \OC_Image::_output(): "image/x-ms-bmp" is not supported when forcing a specific output format	2014-11-23T21:17:56+00:00
Error	PHP	filesize(): stat failed for /var/www/cloud.fsgo.se/owncloud/data/admin/gallery/admin//JV ALLA/Telenor/20052719[1].square.tif at /var/www/cloud.fsgo.se/owncloud/apps/gallery/lib/thumbnail.php#107	2014-11-23T21:17:55+00:00
Error	PHP	filemtime(): stat failed for /var/www/cloud.fsgo.se/owncloud/data/admin/gallery/admin//JV ALLA/Telenor/20052719[1].square.tif at /var/www/cloud.fsgo.se/owncloud/apps/gallery/lib/thumbnail.php#106	2014-11-23T21:17:55+00:00
Error	PHP	imagecreatefromstring(): Data is not in a recognized format at /var/www/cloud.fsgo.se/owncloud/lib/private/image.php#531	2014-11-23T21:17:55+00:00
Fatal	index	Exception: \OC_Image::_output(): "image/x-ms-bmp" is not supported when forcing a specific output format	2014-11-23T21:17:55+00:00
Error	PHP	Undefined index: fileid at /var/www/cloud.fsgo.se/owncloud/lib/private/files/fileinfo.php#90	2014-11-23T20:55:45+00:00
Error	PHP	Undefined index: mimetype at /var/www/cloud.fsgo.se/owncloud/lib/private/files/fileinfo.php#97	2014-11-23T20:55:45+00:00
Error	PHP	Undefined index: fileid at /var/www/cloud.fsgo.se/owncloud/lib/private/files/fileinfo.php#90	2014-11-23T20:54:51+00:00
Error	PHP	Undefined index: mimetype at /var/www/cloud.fsgo.se/owncloud/lib/private/files/fileinfo.php#97	2014-11-23T20:54:51+00:00
Error	PHP	filesize(): stat failed for /var/www/owncloud/data/enoch/gallery/enoch//DirektUppladdning/2011/2011-11-07 18.59.35.jpg at /var/www/owncloud/apps/gallery/lib/thumbnail.php#107	2014-11-23T21:04:17+00:00
Error	PHP	filemtime(): stat failed for /var/www/owncloud/data/enoch/gallery/enoch//DirektUppladdning/2011/2011-11-07 18.59.35.jpg at /var/www/owncloud/apps/gallery/lib/thumbnail.php#106	2014-11-23T21:04:17+00:00
Error	PHP	imagecreatefromjpeg(): '/var/www/owncloud/data/enoch/files/DirektUppladdning/2011/2011-11-07 18.59.35.jpg' is not a valid JPEG file at /var/www/owncloud/lib/private/image.php#464	2014-11-23T21:04:17+00:00
``` 

Get theese errors when looking trhu pictures with the gallery app in the Web UI.

**Comment by oparoz** [comment]
@libasys - Yes, I think it comes from the generic `image`provider which just says it can generate previews of all `image/*` media types, except tiff and gifs

**Comment by enoch85** [comment]
Still getting this in 8.0.3. Any update?

**Comment by oparoz** [comment]
Seems like core can't handle BMP files properly
https://github.com/owncloud/core/issues/16461

**Comment by oparoz** [comment]
This is a problem in core due to the fact that PHP uses a weird media type. Should be fixed in 8.2
